server.js

```diff
@@ -1,20 +1,79 @@
-var http = require('http');
-var secondLargest = require('./programs/secondLargest').secondLargest;
-var calculateFrequency = require('./programs/calculateFrequency').calculateFrequency;
-var flatten = require('./programs/flatten').flatten;
-var unflatten = require('./programs/unflatten').unflatten;
-const util = require('util');
+var http = require("http");
+var secondLargest = require("./programs/secondLargest").secondLargest;
+var calculateFrequency = require("./programs/calculateFrequency")
+  .calculateFrequency;
```

Comment:
@abansal26 '.calculateFrequency' should be in the same line.

Reply:
Changes done as suggested by @sakodiya. Commit ID:-  c1706948e3f496889b54019659dd20ae5148085a